src/components/all-on-one-form-sub-element-button/all-on-one-form-sub-element-button.ts

```diff
@@ -11,37 +11,27 @@ export class AllOnOneFormSubElementButtonComponent {
   @Input() sectionsToShow: string;
   serious: boolean = false;
   dangerous: boolean = false;
+  faultCount: number = 0;
 
   constructor(private faultStore: FaultStoreProvider) {}
 
   ngOnChanges() {
-    const controlKey = 'manoeuver' + this.sectionsToShow + 'Controlv1';
-    const obsKey = 'manoeuver' + this.sectionsToShow + 'Observationv1';
-
     this.faultStore.currentfaults$.subscribe((data) => {
```

Comment:
Is this right? We're subscribing to the faultStore on every change here, I think this will chew memory up.
We only need to do this onInit or the constructor.